robot_skills/src/robot_skills/world_model_ed.py

```diff
@@ -178,8 +179,9 @@ def get_closest_laser_entity(self, type="", center_point=VectorStamped(), radius
             entities = sorted(entities, key=lambda entity: entity.distance_to_2d(
                 center_point.projectToFrame("/%s/base_link" % self.robot_name,
                                             self._tf_listener).vector))  # TODO: adjust for robot
-        except:
+        except Exception as e:
             print("Failed to sort entities")
+            rospy.logerr(e)
```

Comment:
While you're at it, please replace this print statement by a proper logmessage as well